package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "opendistro-kibana-reports",
+  "name": "opendistro_kibana_reports",
   "version": "0.0.1",
   "description": "Kibana Reporting Plugin",
   "license": "Apache-2.0",
-  "main": "index.js",
+  "main": "target/plugins/opendistro_kibana_reports",
```

Comment:
not sure about this part.

Reply:
You are right, changed it to `index.ts`

Reply:
Post the plugin install issue here https://github.com/opendistro-for-elasticsearch/kibana-reports/issues/40